editing/fx/passes/pact/approximate.py

```diff
@@ -57,28 +51,25 @@ class ApproximateSoftmaxPass(SequentialPass):
 
     modes = ["I-BERT", "ITA", 'ITA-Partial']
 
-    def __init__(self, mode: Literal["I-BERT", "ITA", 'ITA-Partial'] = "I-BERT", **kwargs):
+    def __init__(self, symbolic_trace: callable = PACT_symbolic_trace, mode: Literal["I-BERT", "ITA", 'ITA-Partial'] = "I-BERT", **kwargs):
```

Comment:
The type annotation looks wrong. I think it should be Callable[[Union[nn.Module, fx.GraphModule]], fx.GraphModule] or similar.

Reply:
Addressed in [this commit.](https://github.com/pulp-platform/quantlib/pull/15/commits/c2553bef1a29dc0baf6dd19cf1cbd8a2a351cedf)